Yoast/javascript
# Add Czech function words

## Summary


This PR can be summarized in the following changelog entry:

[yoastseo] Adds function words for Czech.
[Yoast SEO Free] Improves keyphrase recognition in Czech by filtering out function words such as `dvou, tvému, nějaký`.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

Test the following in WP SEO Free:

- Set the site language to Czech
- Create a new post and add a text with at least 300 words. To create a text you can use e.g., https://generator.lorem-ipsum.info or use the [Czech Wikipedia](https://cs.wikipedia.org/wiki/Hlavn%C3%AD_strana)
- Set your keyphrase to `čtyři hrady`
- Add the word `hrady` to the text
- Make sure that the keyphrase density shows that the keyphrase is found once

Note: at the moment it's not possible to test this feature in the content-analysis app

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-629
